web/src/lib/components/shared-components/upload-panel.svelte

```diff
@@ -27,30 +27,36 @@
   };
 
   $: $isUploading && autoHide();
+
+  function handleUploadComplete() {
+    if ($errorCounter > 0) {
+      notificationController.show({
+        message: `Upload completed with ${$errorCounter} error${$errorCounter > 1 ? 's' : ''}`,
+        type: NotificationType.Warning,
+      });
+    }
+
+    if ($duplicateCounter > 0) {
+      notificationController.show({
+        message: `Skipped ${$duplicateCounter} duplicate asset${$duplicateCounter > 1 ? 's' : ''}`,
+        type: NotificationType.Warning,
+      });
+    } else if ($successCounter > 0) {
+      notificationController.show({
+        message: 'Upload success, refresh the page to see new upload assets.',
```

Comment:
this doesn't seem like an improvement to me. it seems funny to me that we could have errors and say "upload success"